ui/admin/app/styles/app.scss

```diff
@@ -870,25 +870,9 @@
   }
 }
 
-// Layout - Sidebar
-.layout-sidebar {
```

Comment:
I think we want to review this. If we get rid of all these styles, the sidebar isn't responsive anymore.

Reply:
Small screen issue fixed.